src/document.c

```diff
@@ -533,9 +546,19 @@ FIELD_BULK_INDEXER(numericIndexer) {
       return -1;
     }
   }
-  NRN_AddRv rv = NumericRangeTree_Add(rt, aCtx->doc->docId, fdata->numeric);
-  ctx->spec->stats.invertedSize += rv.sz;  // TODO: exact amount
-  ctx->spec->stats.numRecords += rv.numRecords;
+  
+  if (!fdata->isMulti) {
+    NRN_AddRv rv = NumericRangeTree_Add(rt, aCtx->doc->docId, fdata->numeric, false);
+    ctx->spec->stats.invertedSize += rv.sz;  // TODO: exact amount
```

Comment:
Todo?

Reply:
This TODO is almost 2 years old
I suppose we can just remove it (you added it in PR 1514)